eigrpd/eigrp_zebra.c

```diff
@@ -98,9 +98,7 @@ static zclient_handler *const eigrp_handlers[] = {
 
 void eigrp_zebra_init(void)
 {
-	struct zclient_options opt = {.receive_notify = false};
-
-	zclient = zclient_new(master, &opt, eigrp_handlers,
+	zclient = zclient_new(master, &zclient_options_default, eigrp_handlers,
```

Comment:
Don't we need to add a notify owner call in the connection succeeded function?

Reply:
`.receive_notify = false`

Reply:
ah silly me